frontend/src/components/core/StreamlitDialog/DeployErrorDialogs/DetachedHead.test.tsx

```diff
@@ -0,0 +1,9 @@
+import DetachedHead from "./DetachedHead"
+
+describe("DeployErrorDialog - DetachedHead", () => {
+  it("Should render without crashing", () => {
```

Comment:
Nit: Just to keep the code consistent, prefer starting with lowercase after `it`.